MEAnalysis/crab_vhbb/heppy_crab_script.py

```diff
@@ -44,8 +44,11 @@ def getLumisProcessed(vlumiblock):
     if arg.startswith("ME_CONF="):
         me_conf_name = arg.split("=")[1]
 
-JobNumber=sys.argv[1]
-print "JobNumber", JobNumber
+try:
```

Comment:
with this change, we can do directly:
ME_CONF=cfg_FH.py python heppy_crab_script.py >& logLocal &

What is your cmdline flag ?